project/project_ps.go

```diff
@@ -1,13 +1,23 @@
 package project
 
 import (
+	"sort"
+
 	"golang.org/x/net/context"
 )
 
 // Ps list containers for the specified services.
 func (p *Project) Ps(ctx context.Context, services ...string) (InfoSet, error) {
 	allInfo := InfoSet{}
 	for _, name := range p.ServiceConfigs.Keys() {
+		if services != nil { // apply filter
+			sort.Strings(services)
```

Comment:
This sort could be moved to outside the for loop, right? It might actually be better to just build up the list of needed services prior to this for loop.